pyperformance/data-files/benchmarks/bm_async_tree/bm_async_tree_cpu_io_mixed.toml

```diff
@@ -1,4 +1,3 @@
 [tool.pyperformance]
 name = "async_tree_cpu_io_mixed"
 extra_opts = ["cpu_io_mixed"]
-
```

Comment:
Please keep one trailing line.

Reply:
main has two trailing lines here (and in the other one) - GitHub UI is not super clear about this, but the change here is from two trailing lines to one trailing line (unless my editor is messing with me).